src/actions/productosActions.js

```diff
@@ -1,116 +1,104 @@
 import authAxios from "../config/authAxios";
 import { STATUS_OK } from "../config/constants";
 import response from "../config/network/response";
-import { getCategoriasAsync } from "./categoriasActions";
-import { getMarcasAsync } from "./marcasActions";
 
 export const getProductoByIdAsync = async (idProducto, resolve) => {
-	try {
-		if (!idProducto) {
-			console.error("no product");
-			resolve(null);
-		}
+  try {
+    if (!idProducto) {
+      console.error("no product");
+      resolve(null);
+    }
 
-		console.log("Start request to db");
-		const res = await authAxios.get(`/product/${idProducto}`);
+    const res = await authAxios.get(`/artworks/${idProducto}`);
 
-		console.log("response from db", res);
-		if (res.data.status === STATUS_OK) {
-			resolve(res.data.body);
-		} else {
-			resolve(null);
-		}
-	} catch (error) {
-		resolve(null);
-	}
-};
-
-export const getDataFichaProducto = async (idProducto, resolve) => {
-	try {
-		const res = await Promise.all([
-			getCategoriasAsync(),
-			getMarcasAsync(),
-			getProductoByIdAsync(idProducto, resolve),
-		]);
-
-		return {
-			plato: res[2] ? res[2] : {},
-			marcas: res[1],
-			categorias: res[0],
-		};
-	} catch (error) {
-		console.log(error);
-		response.error(null, "Opps... hubo un error inesperado");
-		return null;
-	}
+    console.log("response from db", res);
+    if (res.status === STATUS_OK) {
+      resolve(res.data.data);
+    } else {
+      resolve(null);
+    }
```

Comment:
It looks good! I would also like to suggest

```suggestion
    if (res.status === STATUS_OK) {
      resolve(res.data.data);
    }
    
    resolve(null);
```

or

```suggestion
    res.status === STATUS_OK ? resolve(res.data.data) : resolve(null);
```